repository/pom.xml

```diff
@@ -208,9 +208,9 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
-      <groupId>org.hitachivantara</groupId>
-      <artifactId>safeyaml</artifactId>
-      <version>${safeyaml.version}</version>
+      <groupId>org.yaml</groupId>
```

Comment:
`Yaml()` is called here: https://github.com/pentaho/pentaho-platform/blob/156fb63fddacda4de89be2f2c7882b7e50f39f62/repository/src/main/java/org/apache/jackrabbit/core/security/authorization/acl/MagicAceDefinition.java#L141